Create ui-c8y-1022-13-1-hide-smart-rules-section-in-custom-microservice-status-tab-if-smartrules.md (#3609)
SoftwareAG/c8y-docs: content/change-logs/platform-services/ui-c8y-1022-13-1-hide-smart-rules-section-in-custom-microservice-status-tab-if-smartrules.md

---
date: ""

> title: Hide Smart rules section in custom microservice Status tab if smartrules microservice is not available

Reviewer: ```suggestion
title: Smart rules section in the Status tab of custom microservices only displayed if Smartrules microservice is available
```

product_area: Platform services
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-0UgqXH1Ys
    label: Administration
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-64173
version: 1022.13.1
---

> In custom microservice "Status" tab, Smart rules section was always visible, even if Smartrules microserivce was unavailable, causing alerts to be thrown. After fix, Smartrules microservice availability is checked before displaying its section and if microservice is not subscribed, section is not displayed at all.

Reviewer 2: ```suggestion
In custom microservice details view ("Status" tab), Smart rules section was always visible, even if Smartrules microserivce was unavailable, causing alerts to be displayed. After fix, Smartrules microservice availability is checked before displaying its section and if microservice is not subscribed, section is not displayed at all.
```

> In custom microservice details view ("Status" tab), Smart rules section was always visible, even if Smartrules microserivce was unavailable, causing alerts to be displayed. After fix, Smartrules microservice availability is checked before displaying its section and if microservice is not subscribed, section is not displayed at all.

Reviewer: ```suggestion
In the **Status** tab of custom microservices, the smart rules section was always visible, even if the Smartrules microservice was unavailable, causing alerts to be displayed. This issue has been resolved. The availability of the Smartrules microservice is now checked, and the smart rules section is only displayed if the Smartrules microservice is available. 
```

Reviewer: @jdre-c8y Is there a difference between "not available" and "not subscribed" in the behavior?

Reviewer 2: technically yes, but on UI I  we cannot tell if microservice is "not available" or "not subscribed" and definitely it does not make difference for user and release note
